### DISCUSSION TITLE
Postgress w dokerze + vm

### DISCUSSION THREAD

**Discussion by szympulka** [AzureCommunityPL/group#417] [Pomoc] [answered]
Hej. Mam bazę w dokerze na vm postawioną i chcę się do niej dobić przez pgadmina. Dodałem port 5432 aby był widoczny publicznie i nadal nie mogę się dobić (dostaje timeout). Czy muszę na vm dodatkowo coś zrobić by wystawić te porty? 

Poniżej docker compose
```  db:
    container_name: aidar_db
    image: postgres:14.1-alpine
    restart: always
    environment:
      PGDATA: /var/lib/postgresql/data/aidar
      POSTGRES_LOGIN: postgres
      POSTGRES_PASSWORD: admin
    ports:
      - '5432:5432'
    volumes:
      - db:/var/lib/postgresql/data:rw
```

**Comment by kamil-mrzyglod** [comment, marked as answer]
Ale co znaczy, że dodałeś port? Zrobiłeś to na poziomie NSG czy samej maszynki?

**Reply by szympulka** [reply]
Na poziomie NSG. Na maszynie odpaliłem tylko docker-compose

**Reply by szympulka** [reply]
Dobra, połączyło nagle. Nie mam pojęcia czemu przedtem nie chciał się połączyć
